.devcontainer/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM mcr.microsoft.com/devcontainers/python:1-3.10-bookworm
+RUN apt-get update \
+    && export DEBIAN_FRONTEND=noninteractive && apt-get install -y libboost-dev \
+    && apt-get clean && rm -rf /var/cache/apt/* && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
```

Can we also add some basic python tooling like `pipx` & `uv`? For example:
```suggestion
    && export DEBIAN_FRONTEND=noninteractive && apt-get install -y libboost-dev \
    && apt-get install pipx && ensurepath && pipx install uv \
    && apt-get clean && rm -rf /var/cache/apt/* && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
```